packages/Liquid-Core.package/LiquidChoicesQuestion.class/instance/type.st

```diff
@@ -0,0 +1,3 @@
+accessing
+type
+	^ 'choices'
```

Comment:
Why do we have this? It is not used anywhere

Reply:
See [Domain model](https://lucid.app/lucidchart/525666f3-ba3c-4665-af03-040fab37f518/edit?shared=true&page=0_0#).
We will have multiple question types (choices, freeText, ...). They will be needed later to display the correct ui.
- Yes this violates "YAGNI", but it is an essential part of the Domain model we already agreed upon and our current assumption is that there will be support for different types of questions -> if we don't stick to the model this could become a very ugly refactoring process to add question types.